### PR TITLE
Update Demo with latest geostyler and SLD 1.1.0 parser

### PR DESCRIPTION
**Pull request by weskamm**
@geostyler/devs please review

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -84,6 +88,7 @@ class App extends React.Component<AppProps, AppState> {
 
   constructor(props: AppProps) {
     super(props);
+    this._sldStyleParserSE.title = 'SLD 1.1.0 - Symbology Encoding';
```
Comment (KaiVolland) [inline src/App.tsx:91]: I think we should solve this is in the parser itself. But this is not blocking this PR.

**Review by KaiVolland — APPROVED** [review]
Nice addition. Thank you.

**Review by jansule — APPROVED** [review]
Very nice!

**Pull request merged**